src/webgpu/api/validation/valid_to_use_with/lost_device/README.txt

```diff
@@ -0,0 +1,5 @@
+Tests of behavior while the device is lost.
```

Comment:
Also we could have the test of the error monad in valid_to_use_with, or do you think it should be in each of the function's tests?

Reply:
I added this, but decided later to remove it (it's more obvious to make it part of the entry point's tests, e.g. createView)